Cargo.toml

```diff
@@ -17,9 +16,3 @@ readme = "README.md"
 description = "Rust X.509 certificate generator"
 repository = "https://github.com/rustls/rcgen"
 keywords = ["mkcert", "ca", "certificate"]
-
-# This greatly speeds up rsa key generation times
-# (only applies to the dev-dependency of rcgen because cargo
-# ignores profile overrides for non leaf packages)
-[profile.dev.package.num-bigint-dig]
```

Comment:
I suppose removing this is okay because we now have aws-lc-rs which can also do rsa keygen.